### ISSUE TITLE
Deleting temp data directory is causing errors in windows

### ISSUE THREAD

**Issue by santiagohdzb** [jprichardson/electron-mocha#49]
Hi everyone, I love this npm module. However, It shows this error on windows:
```
{ [Error: EPERM: operation not permitted, unlink '%localappdata%\Temp\electron-mocha-1458816920476\GPUCache\data_1']
  errno: -4048,
  code: 'EPERM',
  syscall: 'unlink',
  path: '%localappdata%\\Temp\\electron-mocha-1458816920476\\GPUCache\\data_1' }
```
I think this can be solved saving [browserDataPath](https://github.com/jprichardson/electron-mocha/blob/master/index.js#L36) into a file, then opening that file when [`electron.on("exit")`](https://github.com/jprichardson/electron-mocha/blob/master/bin/electron-mocha#L35)

I did that in a fork if you want to look at it: https://github.com/jprichardson/electron-mocha/compare/master...santiagohdzb:master

But probably there's a better way to solve this...

> OS: Windows 10 x64
> node: 5.7
> npm: 3.6.0

**Comment by emmanuelroecker** [comment]
window must be destroy before electron exit
see #56

**Comment by jprichardson** [comment]
Fixed in 1.2.1
